composer.json

```diff
@@ -13,5 +13,10 @@
 		"phpcbf": [
 			"phpcbf -p"
 		]
+	},
+    "require": {
+		"automattic/jetpack-connection": "1.20.0",
```

Comment:
These are not the latest versions.
The latest version of the JP packages has minimum compatibility with WP 5.5 - but we're still testing for 5.4

https://github.com/Automattic/jetpack/pull/17820